auth/pat.go

```diff
@@ -742,22 +373,22 @@ type PATS interface {
 	RevokePATSecret(ctx context.Context, token, patID string) error
 
 	// AddScope function adds a new scope entry.
-	AddPATScopeEntry(ctx context.Context, token, patID string, platformEntityType PlatformEntityType, optionalDomainID string, optionalDomainEntityType DomainEntityType, operation OperationType, entityIDs ...string) (Scope, error)
+	AddScopeEntry(ctx context.Context, token, patID string, scope []Scope) (ScopesPage, error)
 
 	// RemoveScope function removes a scope entry.
-	RemovePATScopeEntry(ctx context.Context, token, patID string, platformEntityType PlatformEntityType, optionalDomainID string, optionalDomainEntityType DomainEntityType, operation OperationType, entityIDs ...string) (Scope, error)
+	RemoveScopeEntry(ctx context.Context, token, patID string, scope []Scope) (ScopesPage, error)
 
```

Comment:
Lets return only error , not scopes or pats
```suggestion
	RemovePATScopeEntry(ctx context.Context, token, patID string, entityType EntityType, optionalDomainID string, operation Operation, entityIDs ...string) (error)
```